lms/templates/bookmarks/bookmarks_results.underscore

```diff
@@ -0,0 +1,42 @@
+<div id="my-bookmarks" class="sr-is-focusable" tabindex="-1"></div>
```

Comment:
This is looking great, @muhammad-ammar .  I noticed you added the "sr-is-focusable"
div.  If you're seeing a dotted line appear above the bookmarks content
when you click the button, don't be concerned.  That bug was fixed here:
https://openedx.atlassian.net/browse/UX-1946

It's fixed on production.